crates/matrix-sdk/src/room/timeline/tests/invalid.rs

```diff
@@ -88,6 +88,9 @@ async fn invalid_event_content() {
         }))
         .await;
 
+    let _read_receipt_update =
+        assert_matches!(stream.next().await, Some(VectorDiff::Set { value, .. }) => value);
+
```

Comment:
I don't think it makes sense for all tests to have read receipt tracking on. Can you make it opt-in at the `TestTimeline` level (i.e. `TestTimeline::new().with_read_receipt_tracking()` or similar)?